models/reviews-models.js

```diff
@@ -2,14 +2,70 @@ const connection = require("../db/connection");
 
 exports.fetchReviewByReviewId = (id) => {
   if (typeof Number(id) === "number" && !isNaN(Number(id))) {
-    if (Number(id) > 0 && Number(id) < 14) {
-      return connection
-        .query(`SELECT * FROM reviews WHERE review_id = $1;`, [id])
-        .then((review) => {
+    return connection
+      .query(
+        `SELECT a.*, COUNT (b.review_id) AS comment_count FROM reviews a
+        LEFT JOIN comments b
+        ON a.review_id = b.review_id
+        WHERE a.review_id = $1
+        GROUP BY a.review_id;`,
+        [id]
+      )
+      .then((review) => {
+        if (review.rowCount > 0) {
           return review.rows[0];
-        });
-    }
-    return Promise.reject({ status: 404, msg: "User Id Not Found" });
+        }
+        return Promise.reject({ status: 404, msg: "Review Id Not Found" });
+      });
   }
-  return Promise.reject({ status: 400, msg: "Invalid User Id" });
+  return Promise.reject({ status: 400, msg: "Invalid Review Id" });
+};
+
+exports.updateReviewByReviewId = (id, update) => {
+  if (update.inc_votes !== undefined && typeof update.inc_votes === "number") {
```

Comment:
```suggestion
  if (update.inc_votes !== undefined && typeof update.inc_votes === "number") {
```
You can probably get rid of this first condition as undefined doesn't have a type of number and won't pass the second condition